keyboards/dz60/rules.mk

```
@@ -20,5 +20,9 @@ BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
 RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
 BLUETOOTH_ENABLE = no       # Enable Bluetooth
 AUDIO_ENABLE = no           # Audio output
+KEY_LOCK_ENABLE = yes		# Keylock mode
 
 LAYOUTS = 60_ansi 60_ansi_arrow 60_ansi_split_bs_rshift 60_hhkb 60_iso 60_abnt2 60_tsangan_hhkb
+
+UNICODE_ENABLE = yes
+DYNAMIC_MACRO_ENABLE = yes
```

These should be moved to the keymap level.
```suggestion

LAYOUTS = 60_ansi 60_ansi_arrow 60_ansi_split_bs_rshift 60_hhkb 60_iso 60_abnt2 60_tsangan_hhkb
```